sgx-types/src/ssa.rs

```diff
@@ -351,6 +351,17 @@ impl Default for StateSaveArea {
     }
 }
 
+impl AsRef<[u8]> for StateSaveArea {
```

Comment:
Is there some way to guarantee in the code that we won't ever try to go in reverse?

Reply:
No. But all such methods are `unsafe`.